.changeset/loud-trainers-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/select': minor
```

Comment:
major change. per react docs wrapping something in fwd ref should be a breaking change

Reply:
Really? I believe you but can you link the docs?